tests/tests/test_state_scripts.py

```diff
@@ -530,12 +530,6 @@
                 "ArtifactInstall_Leave_03",
                 "ArtifactReboot_Enter_01",
                 "ArtifactReboot_Enter_11",
-                "ArtifactReboot_Error_97",
-                "ArtifactReboot_Error_98",
-                "ArtifactRollback_Enter_00",
-                "ArtifactRollback_Enter_01",
```

Comment:
@kacf due to the new client behaviour, these will not be run on the original partition after reboot.